script.js

```diff
@@ -0,0 +1,256 @@
+       let keys = document.querySelectorAll('#keyboard div');
+       if (caps) {
+         element.classList.add('keypress');
+         keys.forEach((el) => {
+           el.innerText = langKeys[i];
+           i++;
+         })
+       } else {
+         isEng == 1 ? langKeys = engKeys : langKeys = rusKeys;
+         element.classList.remove('keypress');
+         keys.forEach((el) => {
+           el.innerText = langKeys[i];
+           i++;
+         });
+       }
+     }
+   });
+  });
+
+  document.addEventListener('keyup', function(event) {
+    if (event.code == 'ShiftLeft') leftShift = false;
+    else if (event.code == 'AltLeft') {
+      if (leftShift) {
+        isEng == 1 ? isEng = 0 : isEng = 1;
+        let l = document.querySelector('.language');
+        isEng == 1 ? l.innerText = 'Eng' : l.innerText = 'Rus';
+        let i = 0;
+        let keys = document.querySelectorAll('#keyboard div');
+        let langKeys;
+        isEng == 1 ? langKeys = engKeys : langKeys = rusKeys;
+        keys.forEach((el) => {
+          el.innerText = langKeys[i];
+          i++;
+        });
+      }
+    }
+    let element = document.querySelectorAll('div[data="' + `${event.keyCode}` + '"]');
+    if (element.length == 1) {
+      element = element[0];
+      element.classList.remove('keypress');
+    } else {
+      if (event.code == "ShiftLeft") element[0].classList.remove('keypress');
+      else if (event.code == "ShiftRight") element[1].classList.remove('keypress');
+      else if (event.code == "ControlLeft") element[0].classList.remove('keypress');
+      else if (event.code == "ControlRight") element[1].classList.remove('keypress');
+      else if (event.code == "MetaLeft") element[0].classList.remove('keypress');
+      else if (event.code == "MetaRight") element[1].classList.remove('keypress');
+      else if (event.code == "AltLeft") element[0].classList.remove('keypress');
+      else if (event.code == "AltRight") element[1].classList.remove('keypress');
+    }
+    if (event.keyCode == 20) {
+      let langKeys;
+      isEng == 1 ? langKeys = engKeysUpperCase : langKeys = rusKeysUpperCase;
+      caps = event.getModifierState && event.getModifierState('CapsLock');
+      let i = 0;
+      let keys = document.querySelectorAll('#keyboard div');
+      if (caps) {
+        element.classList.add('keypress');
+        keys.forEach((el) => {
+          el.innerText = langKeys[i];
+          i++;
+        })
+      } else {
+        isEng == 1 ? langKeys = engKeys : langKeys = rusKeys;
+        element.classList.remove('keypress');
+        keys.forEach((el) => {
+          el.innerText = langKeys[i];
+          i++;
+        });
+      }
+    } else if (event.keyCode == 16) {
+          let langKeys;
+          isEng == 1 ? langKeys = engKeys : langKeys = rusKeys;
+          let i = 0;
+          let keys = document.querySelectorAll('#keyboard div');
+          keys.forEach((el) => {
+            el.innerText = langKeys[i];
```

Comment:
You can create a function to implement this code and reuse it multiple time

Reply:
edited